engine/apps/mobile_app/views.py

```diff
@@ -209,9 +215,9 @@ def _proxy_request(self, request, *args, **kwargs):
         try:
             downstream_response = downstream_request_handler(
                 downstream_url,
-                data=request.data,
+                data=request.body,
```

Comment:
`request.body` = "The raw HTTP request body as a bytestring" ([docs](https://docs.djangoproject.com/en/5.0/ref/request-response/#django.http.HttpRequest.body))

the `requests` library expects:
> data – (optional) Dictionary, list of tuples, bytes, or file-like object to send in the body of the [Request](https://requests.readthedocs.io/en/latest/api/#requests.Request).